### ISSUE TITLE
Support for profiling in PHP 7.0 (XHProf or something else)

### ISSUE THREAD

**Issue by heddn** [geerlingguy/drupal-vm#544]
There are two options it would seem at the moment. The tideway's solution is more popular ATM, but I haven't tested either of them yet.

https://github.com/tideways/php-profiler-extension
https://github.com/RustJason/xhprof/tree/php7

**Comment by geerlingguy** [comment]
I'm still happy with XHProf as long as the extension (or a well-supported fork of it) compiles; it's been the de-facto/baseline for a while now, though others like Blackfire are gaining some traction. See related: #526

**Comment by geerlingguy** [comment]
Also, isn't Tideways for-pay? That's a total non-starter for me... don't want to include any software in Drupal VM that would require a license, for personal and testability reasons.

**Comment by heddn** [comment]
Yeah, I didn't check that far. I just wanted to get some thoughts down in an issue. Using Blackfire seems like a decent option until the forks for XHProf standardize. I'd be worried that those things are in flux a lot right now.

**Comment by geerlingguy** [comment]
I've added Blackfire, and it has 'experimental' support for PHP 7.0 (I haven't personally tested except with 5.6 so far). Hopefully one of the XHProf forks will start adding in solid PHP 7.0 support, but I'm not holding my breath right now.

I'm going to close this for now, as I've also updated the docs with information about the three supported ways to profile code built into Drupal VM, and we can maybe open a follow-up or work on the XHProf issue upstream in that `geerlingguy.php-xhprof` role.
